Doppler.HtmlEditorApi/CampaignState.cs

```diff
@@ -0,0 +1,16 @@
+using System.Linq;
+using System;
+using System.Collections.Generic;
+
+namespace Doppler.HtmlEditorApi;
+
+public record CampaignState(bool OwnCampaignExists, bool ContentExists, int? EditorType, CampaignStatus? CampaignStatus)
+{
+    private static readonly HashSet<CampaignStatus> WritableStatus = new HashSet<CampaignStatus>(
+        new[] {
+            HtmlEditorApi.CampaignStatus.DRAFT,
+            HtmlEditorApi.CampaignStatus.IN_WINNER_IN_AB_SELECTION_PROCESS
+            });
```

Comment:
Very minor

```suggestion
    private static readonly HashSet<CampaignStatus> WritableStatus = new HashSet<CampaignStatus>(new[] {
        Domain.CampaignStatus.DRAFT,
        Domain.CampaignStatus.IN_WINNER_IN_AB_SELECTION_PROCESS
    });

```

Reply:
In my branch I haven't domain namespace. 🤔

Reply:
My comment was related to the code formating, sorry, fixed